module/Decision/src/Decision/Controller/FileBrowser/FileNode.php

```diff
@@ -64,4 +112,30 @@ public function getName()
     {
         return $this->name;
     }
+
+    /**
+     * Extension of file or dir
+     * @return string valid name
+     */
+    public function getExtension()
+    {
+        return $this->extension;
+    }
+
+    /**
+     * Get corresponding FontAwesome file icon
+     * @return string valid FontAwesome 5 icon
+     */
+    public function getFileIcon()
+    {
+
+
```

Comment:
No need for whitespace here.

Reply:
Resolved in 12c64c58ce22698636fce588f88fc82c928d41f9